DIA/MachineType.cs

```diff
@@ -0,0 +1,38 @@
+namespace DIA
+{
+    public enum MachineType
```

Comment:
If names could be in C# style that would be awesome. All enumeration names start with IMAGE_FILE_MACHINE_, that could be omitted and moved to comment and rest of the name could be written as PascalCase.

Reply:
I do agree with this principle actually.
But since CV_CFL_LANG, CV_CPU_TYPE_e and CV_HREG_e did not respect this convention, I did assume that you would prefer it "raw".
I'll will fix it.